Lib/test/test_pathlib.py

```diff
@@ -345,6 +345,8 @@ def test_match_common(self):
         self.assertTrue(P('/a/b.py').match('/a/*.py'))
         self.assertFalse(P('/ab.py').match('/a/*.py'))
         self.assertFalse(P('/a/b/c.py').match('/a/*.py'))
+        # Multi-part glob-style pattern
```

Comment:
```suggestion
        # Multi-part glob-style pattern.
```

Reply:
None of the other single-line comments in this file have periods? I can update all of them or just this one?

Reply:
You can update all of them since this isn't code and it's easy to see what changed.

Reply:
@brettcannon 👍 done in https://github.com/python/cpython/pull/11171/commits/0e2800a4e5c564640650722ccb4641f34dd6b090